<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>patientHealthQuestionClient</class>
 <widget class="QWidget" name="patientHealthQuestionClient">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>879</width>
    <height>568</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>879</width>
    <height>568</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>879</width>
    <height>568</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>健康信息问卷</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
/* 主窗口样式 */
QWidget {
    font-family: &quot;Segoe UI&quot;, &quot;Microsoft YaHei&quot;, sans-serif;
    background-color: #f8fafc;
}

/* 标签样式 */
QLabel {
    color: #374151;
    font-size: 14px;
    font-weight: 500;
    padding: 8px 0px;
    background-color: transparent;
}

/* 输入框样式 */
QLineEdit {
    background-color: white;
    border: 1px solid #e2e8f0;
    border-radius: 8px;
    padding: 10px 12px;
    font-size: 14px;
    color: #1f2937;
    selection-background-color: #4a90e2;
    selection-color: white;
    min-height: 30px;
    max-width: 200px;
}

QLineEdit:focus {
    border: 1px solid #4a90e2;
    background-color: #f0f7ff;
    outline: none;
}

QLineEdit:hover {
    border: 1px solid #cbd5e1;
}

QLineEdit:placeholder {
    color: #9ca3af;
    font-style: italic;
}

/* 下拉框样式 */
QComboBox {
    background-color: white;
    border: 1px solid #e2e8f0;
    border-radius: 8px;
    padding: 8px 12px;
    font-size: 14px;
    color: #1f2937;
    min-height: 30px;
    max-width: 200px;
}

QComboBox:focus {
    border: 1px solid #4a90e2;
    background-color: #f0f7ff;
}

QComboBox:hover {
    border: 1px solid #cbd5e1;
}

QComboBox::drop-down {
    subcontrol-origin: padding;
    subcontrol-position: top right;
    width: 20px;
    border-left: 1px solid #e2e8f0;
    border-top-right-radius: 8px;
    border-bottom-right-radius: 8px;
}

QComboBox::down-arrow {
    image: none;
    border-left: 4px solid transparent;
    border-right: 4px solid transparent;
    border-top: 6px solid #64748b;
}

/* 按钮样式 */
QPushButton {
    background-color: #4a90e2;
    color: white;
    border: none;
    border-radius: 8px;
    padding: 12px 24px;
    font-size: 15px;
    font-weight: 500;
    transition: all 0.2s ease;
}

QPushButton:hover {
    background-color: #3a7bc8;
    transform: translateY(-1px);
}

QPushButton:pressed {
    background-color: #2c6bb6;
    transform: translateY(1px);
}

QPushButton:disabled {
    background-color: #cbd5e1;
    color: #94a3b8;
}

/* 布局容器样式 */
#gridLayoutWidget {
    background-color: white;
    border-radius: 12px;
    border: 1px solid #e2e8f0;
    box-shadow: 0 4px 6px rgba(0, 0, 0, 0.05);
}

/* 表单布局样式 */
QGridLayout {
    margin: 20px;
    spacing: 15px;
}

QVBoxLayout {
    spacing: 10px;
}
   </string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>60</y>
     <width>691</width>
     <height>381</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="2">
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>性别</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>身高 (cm)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>心率 (bpm)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>肺活量 (ml)</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>姓名</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>年龄</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>体重 (kg)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>血压 (mmHg)</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="1">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLineEdit" name="nameLineEdit">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>52</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>226</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="placeholderText">
         <string>请输入姓名</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="ageLineEdit">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>52</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>226</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="placeholderText">
         <string>请输入年龄</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="weightLineEdit">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>52</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>226</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="placeholderText">
         <string>请输入体重</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="pressureLineEdit">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>52</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>226</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="placeholderText">
         <string>例如: 120/80</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="3">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLineEdit" name="genderLineEdit">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>52</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>226</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="placeholderText">
         <string>请输入性别</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="heightLineEdit">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>52</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>226</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="placeholderText">
         <string>请输入身高</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="heartLineEdit">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>52</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>226</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="placeholderText">
         <string>静息心率</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lungLineEdit">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>52</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>226</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="placeholderText">
         <string>请输入肺活量</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>480</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>提交</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
